functions/is-present/1.0/index.js

```diff
@@ -0,0 +1,5 @@
+const isPresent = async ({ record: { data } }) => ({
+  result: !!data,
```

Comment:
This does not properly check if the data is present. If the value is `false` it will return false, but should return true in this case. Also the case for empty strings if value is 0. Maybe easier to check for `null`, `undefined`?

Reply:
Also test all the cases you want to check on in your tests